plugins/chat/assets/javascripts/discourse/components/channels-list-direct.gjs

```diff
@@ -0,0 +1,215 @@
+import Component from "@glimmer/component";
```

Comment:
I don't know if that would still be necessary once we sort the scrolling question (more on this later) but ATM, we should try to come up with the way to not duplicate the whole scrollbar logic in two components.